autoware_map_msgs/srv/DifferentialPointCloudMap.srv

```diff
@@ -0,0 +1,13 @@
+# Area query for map loading
+# (PCD grids that overlaps with the area is going to be loaded)
+AreaInfo area
+
+# The IDs of PCD maps that the node already has
+string[] already_loaded_ids
+
+---
+# Newly loaded PCD maps with ID
+PointCloudMapWithID[] loaded_pcds
```

Comment:
[same comment](https://github.com/autowarefoundation/autoware_msgs/pull/39/files#r981952446)
example:
- additional_pointcloud
- additional_pointcloud_with_id
- new_pointcloud
- new_pointcloud_with_id

Reply:
BTW is "pcd" that unfamiliar? I thought it is pretty common words, given that the extension of the data is ".pcd".